Userland/Libraries/LibWebView/WebContentClient.cpp

```diff
@@ -34,13 +34,6 @@ void WebContentClient::unregister_view(u64 page_id)
     m_views.remove(page_id);
 }
 
-void WebContentClient::notify_process_information(WebView::ProcessHandle const& handle)
```

Comment:
fwiw the set_process_information hook was there mostly for serenity, where the process is spawned by SystemServer.